### PR TITLE
[Docs] Add broker additional servlet configuration

### PR DESCRIPTION
**Pull request by Demogorgon314**
Add docs for broker additional servlet (https://github.com/apache/pulsar/pull/11498).

### CODE REVIEW

File: site2/docs/reference-configuration.md [hunk 1]
```diff
@@ -349,7 +349,9 @@ brokerServiceCompactionThresholdInBytes|If the estimated backlog size is greater
...
-/ managedLedgerInfoCompressionType / ManagedLedgerInfo compression type, option values (NONE, LZ4, ZLIB, ZSTD, SNAPPY), if value is NONE or invalid, the managedLedgerInfo will not be compressed. Notice, after enable this configuration, if you want to degrade broker, you should change the configuration to `NONE` and make sure all ledger metadata are saved without compression. | None |
+| managedLedgerInfoCompressionType | ManagedLedgerInfo compression type, option values (NONE, LZ4, ZLIB, ZSTD, SNAPPY), if value is NONE or invalid, the managedLedgerInfo will not be compressed. Notice, after enable this configuration, if you want to degrade broker, you should change the configuration to `NONE` and make sure all ledger metadata are saved without compression. | None |
+| additionalServlets | The additional servlet name, multiple additionalServlets are separated by commas. For example: additionalServlet_1, additionalServlet_2 | N/A |
+| additionalServletDirectory | Location of broker additional servlet NAR Directory | ./brokerAdditionalServlet |
```
Comment (Anonymitaet) [inline site2/docs/reference-configuration.md:354]:
```suggestion
| managedLedgerInfoCompressionType | Compression type of managed ledger information. <br><br>Available options are `NONE`, `LZ4`, `ZLIB`, `ZSTD`, and `SNAPPY`). <br><br>If this value is `NONE` or invalid, the `managedLedgerInfo` is not compressed. <br><br>**Note** that after enabling this configuration, if you want to degrade a broker, you need to change the value to `NONE` and make sure all ledger metadata is saved without compression. | None |
| additionalServlets | Additional servlet name. <br><br>If you have multiple additional servlets, separate them by commas. <br><br>For example, additionalServlet_1, additionalServlet_2 | N/A |
| additionalServletDirectory | Location of broker additional servlet NAR directory | ./brokerAdditionalServlet |
```

Comment (Anonymitaet) [reply site2/docs/reference-configuration.md:354]: Do you want to mean this?

**Review by Anonymitaet — COMMENTED** [review]
Thanks for your contribution. I’ve left some comments, PTAL.

File: site2/docs/reference-configuration.md (hunk 1 above)
Comment (Demogorgon314) [reply site2/docs/reference-configuration.md:354]: Right, but in `managedLedgerInfoCompressionType `, I see a markdown syntax error，just changed the `/` to `|` .

**Review by Anonymitaet — APPROVED** [review]

**Review by codelipenghui — APPROVED** [review]

**Pull request merged**